invokeai/app/services/generate_initializer.py

```diff
@@ -1,16 +1,17 @@
 import os
 import sys
+import torch
 import traceback
 from argparse import Namespace
+from omegaconf import OmegaConf
 
 import invokeai.version
-from invokeai.backend import Generate, ModelManager
-
+from ...backend import ModelManager
+from ...backend.util import choose_precision, choose_torch_device
 from ...backend import Globals
 
-
 # TODO: most of this code should be split into individual services as the Generate.py code is deprecated
-def get_generate(args, config) -> Generate:
+def get_model_manager(args, config) -> ModelManager:
```

Comment:
is `config` even used?

Reply:
No, and I've now gotten rid of it. It is actually the same as `args` except that `args` represents the merge of the command-line options and the init file options, and `config` is just the command-line args. To add to the confusion, the names were reversed in the code.